clients/ui/frontend/src/app/utils/const.ts

```diff
@@ -0,0 +1,9 @@
+export const POLL_INTERVAL = process.env.POLL_INTERVAL
```

Comment:
we have `POLL_INTERVAL` in shared library.
 `FAST_POLL_INTERVAL` and `SERVER_TIMEOUT` are not used in codebase, so I think we can remove this file